In [1]:
import pandas as pd

In [31]:
t1=pd.read_csv("Poor Goalkeeper.csv")
temp=t1['Position'].unique()
t=[]

for i in temp:
    count=0
    for j in range(len(t1)):
        if t1.iloc[j,10]==i:
            count+=1
    t.append([i,count])
t

[['GK', 881]]

Class ->
                Actuall        False
Defender ->     1213           Mid (219) 
Mid->           1024             Def (550) , For(72)
For ->          909            Mid(715)  , Def (5)
Gk->            881              - 